back/engines/commercial/multi_tenancy/spec/services/side_fx_app_configuration_spec.rb

```diff
@@ -10,32 +10,39 @@
 
   it 'has an #before_update method' do
     # only test for the presence of the method bc it does nothing for now.
-    service.before_update(AppConfiguration.instance, current_user)
+    service.before_update(config, current_user)
   end
 
-  describe 'after_update' do
+  describe '#after_create' do
+    it 'enqueues an Seo::UpdateGoogleHostJob job' do
+      expect { service.after_create(config, current_user) }.to enqueue_job(Seo::UpdateGoogleHostJob)
+    end
+  end
+
+  describe '#after_update' do
     it "logs a 'changed' action job when the configuration has changed" do
       settings = config.settings
       settings['core']['organization_name'] = { 'en' => 'New name' }
+
       config.update!(settings: settings)
 
       expect { service.after_update(config, current_user) }
-        .to  have_enqueued_job(LogActivityJob).with(config, 'changed', current_user, config.updated_at.to_i, {})
-        .and have_enqueued_job(LogActivityJob).with(tenant, 'changed', current_user, config.updated_at.to_i, {}) # TODO_MT to be removed
+        .to  enqueue_job(LogActivityJob).with(config, 'changed', current_user, config.updated_at.to_i, {})
+        .and enqueue_job(LogActivityJob).with(tenant, 'changed', current_user, config.updated_at.to_i, {})
     end
 
     it "logs a 'changed_lifecycle_stage' action job when the lifecycle has changed" do
       settings = config.settings
       old_lifecycle_stage = settings.dig('core', 'lifecycle_stage')
       settings['core']['lifecycle_stage'] = 'churned'
+
       config.update!(settings: settings)
-      options = { payload: { changes: [old_lifecycle_stage, 'churned'] } }
 
+      options = { payload: { changes: [old_lifecycle_stage, 'churned'] } }
       updated_at = config.updated_at.to_i
       expect { service.after_update(config, current_user) }
-        .to  have_enqueued_job(LogActivityJob).with(config, 'changed_lifecycle_stage', current_user, updated_at,
-          options)
-        .and have_enqueued_job(LogActivityJob).with(tenant, 'changed_lifecycle_stage', current_user, updated_at, options) # TODO_MT to be removed
+        .to  enqueue_job(LogActivityJob).with(config, 'changed_lifecycle_stage', current_user, updated_at, options)
+        .and enqueue_job(LogActivityJob).with(tenant, 'changed_lifecycle_stage', current_user, updated_at, options)
```

Comment:
Hmm, so I was wrong?

Reply:
Every time the app-config sidefx service logs an activity, it also logs an equivalent activity (with the item equals to the tenant). As a result, these tenant activities are currently being logged twice, both by the app-config sidefx service and the tenant sidefx service. We are simply removing the duplication here.

Reply:
Ok, got it